generators/generator-bot-core-assistant/README.md

```diff
@@ -1,48 +1,34 @@
 # @microsoft/generator-bot-core-assistant [![NPM version][npm-image]][npm-url]
 
-This template creates an assistant-style conversational bot. Assistant-style bots typically help their users accomplish multiple different tasks, and have support for more varied conversational interactions.
+A bot with Azure Language Understanding (LUIS) and common trigger phrases used to direct the conversation flow and help customers accomplish basic tasks.
 
-Includes support for:
+### Recommended use
+
+- Create a sophisticated conversational bot
+- Customize and extend sophisticated dialogs, bot logic, language understanding and bot responses
+
+### Included capabilities
 
 - Greeting new and returning users
+- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
 - Asking for help
-- Cancelling a dialog
-- Submitting feedback about the bot
 - Error handling in conversations
+- Cancelling a dialog
+- Get customer feedback
 - Repeat the previous question
-- Chit chat with QnA Maker ([professional personality](https://docs.microsoft.com/en-us/azure/cognitive-services/qnamaker/how-to/chit-chat-knowledge-base?tabs=v1))
 - Disambiguation of NLP results
```

Comment:
nit: NLP is an acronym not defined elsewhere in the README, in fact the "duplicated intents recognized" documentation doesn't reference "NLP" either. I suggest "Disambiguation when multiple intents are recognized"

Reply:
updated core assistant and ent assistant readme's to use the suggested disambiguation text